sched_slack_bot/model/schedule.py

```diff
@@ -148,7 +148,7 @@ def from_modal_submission(cls, submission_body: SlackBody) -> Schedule:
 
         schedule_id = submission_body["view"]["external_id"]
 
-        if schedule_id == CREATE_NEW_SCHEDULE_VIEW_ID:
+        if schedule_id.startswith(CREATE_NEW_SCHEDULE_VIEW_ID):
             schedule_id = str(uuid.uuid4())
 
         return Schedule(
```

Comment:
I think you're passing an idea with a bad schedule_id here in the non-CREATE_NEW_SCHEDULE_VIEW_ID case.

Reply:
I tried to fix it by adding a fix delimiter: thus I can distinguish the objectid and view id: so ideally it would look like 
external_id = (objectId::uuid)

Reply:
you could also just add the postfix uuid in the case of CREATE_NEW_SCHEDULE_ID ? so it would be either

```CREATE_NEW-{uuid}``` or ```{schedule-id-to-edit-id}``` not sure what is cleaner, both is fine for me

Reply:
Editing existing scheduler also raises the duplicate_external_id exception. So I believe we _always_ need the added uuid

Reply:
it doesn't fully make sense to me tbh, this means someone ALWAYS has an edit schedule dialog open???

Reply:
maybe they write every view that is ever opened to some DB with a uniqueness constraint? weirder things have happened before 😬